srv/salt/ceph/mds/shutdown.sls

```diff
@@ -1,3 +1,3 @@
 shutdown daemon:
   service.dead:
-    - name: ceph-mds@{{ grains['host'] }}
+    - name: ceph-mds@mds.{{ grains['host'] }}
```

Comment:
Shouldn't this be using `mds.get_name`?

Reply:
Right, good catch. I also fixed the restart states.